internal/validator/helpers.go

```diff
@@ -14,8 +14,20 @@ const (
 	BeefHex
 )
 
+type FindSourceFlag byte
+
+const (
+	SourceTransactionHandler = 1
+	SourceNodes              = 2
+	SourceWoC                = 4
+)
```

Comment:
I see this is binary. I saw this way of doing it:
```suggestion
const (
	SourceTransactionHandler = 1 << iota // 1 (binary: 001)
	SourceNodes                                          // 2 (binary: 010)
	SourceWoC                                             // 4 (binary: 100)
)
```
Maybe this will be better so someone doesn't change `4` to `3` because that was my initial thought 😄  WDYT?

Reply:
Good point! Changed